dev-python/flatbuffers/flatbuffers-23.1.21.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..11} )
+
+inherit distutils-r1
+
+DESCRIPTION="RFC 7049 - Concise Binary Object Representation"
+HOMEPAGE="
+	https://github.com/google/flatbuffers/
+	https://pypi.org/project/flatbuffers/
+"
+SRC_URI="
+	https://github.com/google/${PN}/archive/v${PV}.tar.gz
+		-> ${P}.gh.tar.gz
+"
+S=${WORKDIR}/${P}/python
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="amd64 arm arm64 ~riscv x86"
```

Comment:
```suggestion
KEYWORDS="~amd64 ~arm ~arm64 ~riscv ~x86"
```

Reply:
This version has been stabilized before: https://bugs.gentoo.org/893044.
I'm not sure if I should add it back as unstable or not.

Reply:
I think it's fine to add the stable keywords in that case.